src/ads.js

```diff
@@ -86,10 +86,17 @@ export default function getAds(player) {
       player.ads._state.skipLinearAdMode();
     },
 
+    // With no arguments, returns a boolean value indicating whether or not
+    // contrib-ads is set to treat ads as stitched with content in a single
+    // stream.
+    // With an argument, turns stitched ads mode on or off. This is best used
+    // immediately preceding a source change where the state would change to
+    // or from stitched ads.
```

Comment:
Is it the same for the initial source, that it should be called before a source is set?

Reply:
Yeah. There's a change in `plugin.js` that moves the setting of this to before the initial state is created. Otherwise, contrib-ads starts with `stitchedAds` being `false`.